airflow/contrib/hooks/gcp_video_intelligence_hook.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from google.cloud.videointelligence_v1 import VideoIntelligenceServiceClient
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+class CloudVideoIntelligenceHook(GoogleCloudBaseHook):
+    """
+    Hook for Google Cloud Video Intelligence APIs.
+    """
+
+    _conn = None
+
+    def __init__(self, gcp_conn_id="google_cloud_default", delegate_to=None):
+        """
+        :param gcp_conn_id: The connection ID to use when fetching connection info.
```

Comment:
You should choose one place for class documentaiton . Otherwise, it will not generate correctly.

Reply:
Moved to docstring underneath class name